adapters/vidazoo/vidazoo.go

```diff
@@ -0,0 +1,140 @@
+package vidazoo
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+// Builder builds a new instance of the Foo adapter for the given bidder with the given config.
+func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+	bidder := &adapter{
+		endpoint: config.Endpoint,
+	}
+	return bidder, nil
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+
+	// Marshal the OpenRTB request to JSON
+	openRTBRequestJSON, err := json.Marshal(request)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("marshal bidRequest: %w", err))
+		return nil, errs
+	}
+
+	// Extract CID from the request
+	cId, err := extractCid(request)
+	if err != nil {
+		errs = append(errs, fmt.Errorf("extract cId: %w", err))
+		return nil, errs
+	}
+
```

Comment:
based on `extractCid` function implementation, it seems like `cId` is expected to be same for all `imps`.

if so then should specify above behaviour in bidder docs.

Reply:
Changed the logic to imp splitting